src/frontend/src/flows/register/finish.ts

```diff
@@ -18,21 +24,28 @@ const pageContent = (userNumber: bigint) => html`
       ${userNumber}
     </div>
     <div class="l-stack">
-      <button id="displayUserContinue" class="c-button">Continue</button>
+      <button
+        @click=${() => onContinue()}
```

Comment:
nth: maybe you can even simplify here with `@click=${onContinue}` - not sure, to be tested